### ISSUE TITLE
Error 502 Bad Gateway

### ISSUE THREAD

**Issue by schpvs92** [Checkmk/grafana-checkmk-datasource#265]
Hey guys,

i got the following issue:
using Grafana v11.2.0 (c57667e448)
grafana-checkmk-datasource 3.3.0
and
checkmk RAW 2.2.0p9

Im getting the error 502 Bad Gateway when im trying to test the grafana-checkmk-datasource connection to my checkmk instance.
The Firewall isnt blocking, grafana is running in a docker.

Does anyone have any idea what might be the issue?

**Comment by schpvs92** [comment]
@BenediktSeidl hey. thanks for the fast answer.

Pinging from docker works fine. Im also able to see the requests in our firewall.
Actually by using
docker logs grafana i found the following error:
 level=error msg="Proxy request failed" err="tls: failed to verify certificate: x509: cannot validate certificate for "IP of our checkmk server" because it doesn't contain any IP SANs"

**Comment by BenediktSeidl** [comment]
So you are using a self-signed certificate, but Grafana or the docker container does not trust the CA? Or something similar?

**Comment by schpvs92** [comment]
Yes, it seems like grafana is not trusting the CA. To check that, is it possible to disable that security mechanism?

**Comment by BenediktSeidl** [comment]
I don't know, this is independent of the checkmk data source, we just use the Grafana API for this.

But you can use something like https://mitmproxy.org/ and use a `http://` URL for the data source instead of a `https://` one.
